gnucash/gnome/window-reconcile.c

```diff
@@ -382,11 +382,19 @@ gnc_start_recn_date_changed (GtkWidget *widget, startRecnWindowData *data)
 {
     GNCDateEdit *gde = GNC_DATE_EDIT (widget);
     gnc_numeric new_balance;
-    time64 new_date;
+    time64 new_date, cutoff_date;
 
     if (data->user_set_value)
         return;
     new_date = gnc_date_edit_get_date_end (gde);
+    cutoff_date = gnc_time64_get_today_end () + (7*24*60*60);
```

Comment:
I agree with Dale Phurrough that an arbitrary cut-off date is hackish. How about adding another date picker to the reconcile info dialog so that the user can decide?

Reply:
> I agree with Dale Phurrough that an arbitrary cut-off date is hackish. How about adding another date picker to the reconcile info dialog so that the user can decide?

I don't understand what a second date-picker would do here?   The point is to prevent the user from accidentally choosing a day WAY in the future.